### DISCUSSION TITLE
Tesseract 5.1.0 cannot recognize music symbol

### DISCUSSION THREAD

**Discussion by voska7** [SubtitleEdit/subtitleedit#6030] [Q&A]
it looks like tesseract 5.1.0 cannot recognize music symbol, is there anyway to fix it?

https://i.postimg.cc/kGh5d2tk/image.png

**Comment by niksedk** [comment]
Try getting some Tesseract folks include music symbol: https://github.com/tesseract-ocr/tesseract/issues

Alternatively, try nOCR

**Comment by voska7** [comment]
https://www.nikse.dk/subtitleedit/nocr

you suggested:

> Conclusion
> nOCR is a good choice for Blu-ray .sup files with a standard-like-font. 


also again you suggested

>  If the font size is small (letter height less than ~28 pixels), you should use "Tesseract 5" or "Binary image compare".
> You can double click on a line in the list view to start the "Inspect window" where you can see how letters will be split, recognized and their sizes.
> 
> "Number of pixels is space" will determine how words will be found, so "h o w a r e y o u" indicates a too low value, and "howareyou" indicates a too high value. For a normal Blu-ray .sup file 8-15 pixels of space between words are common, 10-12 for most. 


can you please tell what is the difference between "binary image" and "nOCR"

**Reply by niksedk** [reply]
`binary image ocr` is a whole image pixel compare - very tuned to the actual image size, so it works best of all letter are rendered exactly the same everytime and not too big (like many old DVDs).

nOCR is vector/line based, so it does not require image sizes to be exactly the same or the rendering to be exactly the same every time.
Some trained letter can have poor vectors/lines saved, so these may need to be improved/deleted.
